website_sale_vat_required/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,5 @@
+* Lorenzo Battistini <lorenzo.battistini@agilebg.com>
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+    * Jairo Llopis <jairo.llopis@tecnativa.com>
+    * Ernesto Tejeda <ernesto.tejeda87@gmail.com>
```

Comment:
You shouldn't put your personal e-mail here. In fact, we are even removing our corporate ones in all places. I'm changing this myself if the rest is correct.